src/extension.ts

```diff
@@ -231,3 +240,63 @@ async function notifyEditorServiceDisabled(context: vscode.ExtensionContext) {
     context.globalState.update(suppressEditorServicesDisabled, true);
   }
 }
+
+async function notifyIfNewPDKVersion(context: vscode.ExtensionContext, settings:IAggregateConfiguration) {
+  const suppressPDKUpdateCheck = 'suppressPDKUpdateCheck';
+  const dontCheckAgainNotice = "Don't check again";
+  const viewPDKDownloadPage = "More info";
+
+  if (context.globalState.get(suppressPDKUpdateCheck, false)) {
+    return;
+  }
+
+  let version = '1.12.0';
```

Comment:
I'm guessing this shouldn't be hardcoded?

Reply:
Nope. Stupid mistake. fixed